setup.py

```diff
@@ -14,7 +14,7 @@
     name='taskgraph',
     use_scm_version={'version_scheme': 'post-release',
                      'local_scheme': 'node-and-date'},
-    setup_requires=['setuptools_scm'],
+    setup_requires=['setuptools_scm', 'rstcheck'],

```

Comment:
Since `rstcheck` is only used by the tests and not by `setup.py` itself, shouldn't this go into `requirements.txt`?

Reply:
See my comment below about the same thing!